README.md

```diff
@@ -687,31 +683,28 @@ in case the DB is ever "compromised"
 the "attacker" still has to "compute"
 a ["rainbow table"](https://en.wikipedia.org/wiki/Rainbow_table) from _scratch_.
 
-> _**Note**: Don't forget to export your_ `SECRET_KEY_BASE`
-_environment variable_ (_see instructions above_)
+> _**Note**: Don't forget to export your_ `SECRET_KEY_BASE` > _environment variable_ (_see instructions above_)
 
 > The full file containing these two functions is:
-[`lib/encryption/hash_field.ex`](https://github.com/dwyl/phoenix-ecto-encryption-example/blob/master/lib/encryption/hash_field.ex) <br />
+> [`lib/encryption/hash_field.ex`](https://github.com/dwyl/phoenix-ecto-encryption-example/blob/master/lib/encryption/hash_field.ex) <br />
 > And the tests for the functions are:
-[`test/lib/hash_field_test.exs`](https://github.com/dwyl/phoenix-ecto-encryption-example/blob/master/test/lib/hash_field_test.exs)
-
-
+> [`test/lib/hash_field_test.exs`](https://github.com/dwyl/phoenix-ecto-encryption-example/blob/master/test/lib/hash_field_test.exs)
 
 #### 3.5 Hash _Password_
 
 When hashing **passwords**, we want to use the **_strongest_ hashing algorithm**
-and we also want the hashed value (_or "digest"_) to be ***different****
+and we also want the hashed value (_or "digest"_) to be **\*different\*\***
```

Comment:
I suspect that the change here is `***different****` to `***different****` (_the typo is an extra asterisk after he word_) no need for any character escaping (_back slashes_)